tests/results/TextPositionTest.py

```diff
@@ -36,23 +36,54 @@ def test_comparison(self):
         uut2 = TextPosition(None, None)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 > uut2)
+        self.assertFalse(uut1 < uut2)
 
         uut1 = TextPosition(1, 2)
         uut2 = TextPosition(1, 3)
+        uut3 = TextPosition(1, None)
+        uut4 = TextPosition(1, 5)
         self.assertFalse(uut1 >= uut2)
+        self.assertTrue(uut2 > uut1)
+        self.assertFalse(uut3 > uut4)
         self.assertTrue(uut1 <= uut2)
+        self.assertTrue(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
 
         uut1 = TextPosition(2, None)
         uut2 = TextPosition(3, None)
         self.assertFalse(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 > uut2)
+        self.assertTrue(uut1 < uut2)
 
         uut1 = TextPosition(None, None)
         uut2 = TextPosition(4, 5)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut1 <= uut2)
+        self.assertFalse(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
 
         uut1 = TextPosition(4, 8)
         uut2 = TextPosition(4, 8)
         self.assertTrue(uut1 >= uut2)
         self.assertTrue(uut2 <= uut1)
+        self.assertFalse(uut1 > uut2)
+        self.assertFalse(uut2 < uut1)
+
+        uut1 = TextPosition(1, 2)
+        uut2 = TextPosition(4, 3)
+        self.assertTrue(uut1 < uut2)
+        self.assertFalse(uut1 > uut2)
```

Comment:
Remember, how you once told me that I should be a politician? I can now safely say: You too. :grin: